src/app/shared/utils/consts.ts

```diff
@@ -70,7 +70,8 @@ export const Consts = {
         'gcp-file' : 'GCP File Storage',
         'ibm-cos': "IBM COS",
         'yig': "YIG Ceph",
-        'alibaba-oss' : "Alibaba Object Storage"
+        'alibaba-oss' : "Alibaba Object Storage",
+        'sony-oda':"Sony ODA"
```

Comment:
As per Araki-san 
> It is necessary to use "Sony" rather than "SONY".
> And it is preferable to use "Optical Disc Archive" than "ODA". Then I think "Sony Optical Disc Archive" is the best.

Let's call it `Sony Optical Disc Archive`.

Reply:
Updated to Sony Optical Disc Archive